data/json/items/comestibles/drink.json

```diff
@@ -1008,6 +1008,18 @@
     "flags": [ "EATEN_HOT" ],
     "fun": 6
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_willowbark",
+    "copy-from": "tea",
```

Comment:
From the tests/comestible_test.cpp down below - calories need to be set to 0, apparently?